fixed mongoDb metadata config for Post Document #371

codebach opened the pull request:
I am targeting this branch, because this does not work on this branch regarding to dependencies. NOT: This is a BC break but this needs to be fixed.

Closes #371 

## Changelog
```markdown
### Fixed
- Fixed `Post` Document mongoDb metadata from `comments_count` to `commentsCount`
```

## To do
- [x] Add an upgrade note

## Subject

Can you please let me know how to update Docs or add Notes?

greg0ire commented:
Updating the docs is not necessary in this case I think, but an upgrade note might be if you think this worked somehow for some people.

codebach commented:
> Updating the docs is not necessary in this case I think, but an upgrade note might be if you think this worked somehow for some people.

@greg0ire Although I am not sure if this is working for some people, I added an upgrade note to let them know to update their collections in case of problems. I didn't know which version it will be though, so I added under the title: `UPGRADE FROM 3.1.2 to 3.x`

Will it be a new release after this is merged?

core23 commented:
Changing a fieldname in MongoDB is BC?

Soullivaneuh commented on UPGRADE-3.x.md:
```diff
@@ -1,6 +1,11 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.1.2 to 3.x
```
Please do not add upgrade title. Just add the note on the `3.x` section.

codebach replied:
@Soullivaneuh done.

Soullivaneuh commented:
Okay for the upgrade note on this specific case.

codebach commented:
> Changing a fieldname in MongoDB is BC?

@core23 If you have some Collections saved with old attribute, then I thought it is a BC.. I might be wrong though.

codebach commented:
Is there anything else I can do? If not can we please merge this and make a new release?

codebach commented:
Thanks for your approval @Soullivaneuh 

> Looks good, but I would like to have an approval from someone using this bundle.

This is a lock for me to upgrade to Sonata 3. How long do you think this will take?

greg0ire commented:
> How long do you think this will take?

ages. Let's be realistic and merge.

codebach commented:
> ages. Let's be realistic and merge.

Thank you @greg0ire

greg0ire commented:
I'll ask for a release, @Soullivaneuh does them

codebach commented:
Hi guys.. Any idea when we can have a new release for this bundle?

codebach commented:
Any idea about when we can get a new release?

greg0ire commented:
I'll ask again I guess

Soullivaneuh commented:
@codebach Sorry for that, I drunk too much :wine_glass: while making the release and did wrong.

It's fixed now: https://github.com/sonata-project/SonataNewsBundle/releases/tag/3.2.0

codebach commented:
> @codebach Sorry for that, I drunk too much 🍷 while making the release and did wrong.

Ahaha, Well, I hope you enjoyed it, rest is not important :D

Thanks for the release..
